proj.4/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 ./configure --prefix=$PREFIX --without-jni
+
 make
+
+if [[ $(uname) == Linux ]]; then
+    make check
+fi
```

Comment:
> For 4.9.2 as well?

Don't remember, but I had other issues building cartopy on Windows against Proj.4 4.9.2.

> What is the error?

https://travis-ci.org/ioos/conda-recipes/builds/84817022

> We should push that up to proj.4.

Not sure what you mean.